dashboards/cloud-armor/metadata.yaml

```diff
@@ -0,0 +1,11 @@
+sample_dashboards:
+  -
+    category: Cloud Armor
+    id: cloud-armor-policies-overview
+    display_name: Cloud Armor Policies Overview
+    description: |-
+      This dashboard is based on Google Cloud's Cloud Armor Policies (also known as Network Security Policies) and has charts displaying Requests for each policy, as well as a table of open incidents related to your Cloud Armor Policies.
+      There are also links to Cloud Armor Policies in Google Cloud Platform and docs for Observing your Cloud Armor Policies.
+    related_integrations:
+      - id: cloud-armor
```

Comment:
Does this exist? Let's remove this or hold on submitting until it exists.

Reply:
It is being added, let's hold until it's submitted.

Reply:
Cloud Armor integration has been added